WTWSMS/decisions/realm_decisions.txt

```diff
@@ -2028,26 +2061,6 @@ decisions = {
 					value = off
 				}
 			}
-			
-			OR = {
-				ai = no
-				AND = {
-					trait = proud
-					prestige = 5000
-				}
-				AND = {
-					trait = ambitious
-					prestige = 5000
-				}
-				AND = {
-					realm_size = 70
-					any_demesne_title = {
-						count = 4
-						tier = DUKE
-					}
-				}
-				prestige = 15000
-			}
```

Comment:
Either lax the de jure so that AI can create titles fairly, or allow the AI to create titles this way. Anything else is completely neuters the AI in several cases. 

The Empire one is the worst: not even players can normally get that much Prestige until the very end of the game. I played a seven hundred year marathon, and even at the end, I barely reached that level, ever.

Also, creating custom realms is not blobbing. It can only be taken if the AI blobs on its own. It doesn't make the AI blob. Rather, if he does it on his own accord, he can keep that blob under the right circumstances.